go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/mholt/archives v0.1.0
 	github.com/mongodb-forks/digest v1.1.0
 	github.com/mongodb-labs/cobra2snooty v0.18.2
-	github.com/mongodb/mongodb-atlas-kubernetes/v2 v2.6.1
+	github.com/mongodb/mongodb-atlas-kubernetes/v2 v2.6.2-0.20250123164100-b8aa5669900f
```

Comment:
i'd expect 2.7 here

Reply:
Yes, this is the pending part that was waiting for the release.
Branch will be rebased and dependency will be bumped